content.md

```diff
@@ -1,125 +1,126 @@
-	- **Recruited diverse team of module instructors/TAs** from the office, so that the students would see themselves reflected in their instructors.
-	- **Led instructional/pedagogical training** for module instructors and TAs.
-	- Provided **personalized mentorship** to students.
-- Received **4.9/5.0 instructor rating** from students end-of-program survey, as well as the **Google LAX Citizenship Award** and **Google Connect Impact Award**.
-- In 2019, **91% of students reported increased sense of belonging** in the field, and **85% said the program helped prepare them for college**.
-
-#### Python Module Instructor - Summer 2017
-- **Planned and taught** two full-day (9am-6pm) lessons on Python to the students, as well TAing frequently.
-
-Skills
-------
-
-#### Expert in:
-- Languages (computer): **Python**, **Bash** (sad, but true)
-- Languages (human): **English** (written and spoken!)
-- Large-scale system management:
-  - **Large-Scale System Design**
-  - **Monitoring/alerting**
-  - **Reliable deployment**
-- **Linux** system administration
-
-#### Enthusiastic dilettante in:
-- Languages (computer): **JavaScript/ECMAScript**, **Node.js**, **Go**
-- Languages (human): **Spanish**
-- **Teaching** peers and young students.
-
-#### It’s been a while, but I could pick it back up:
-- Languages (computer): **C**, **C++**, **Java**, **Ruby** (on or off Rails), **Perl**, **Lisp**
-- Languages (human): **French**, **Hindi**, **Bengali**
-
-
-Education
----------
-
-### University of Virginia, Dept of Computer Science
-#### M.Cs. in Computer Science - Jan 2008
-Thesis project: **“Color CT: Capturing Energy Dependence in CT Scans”**  
-Publications: Williams, Daniel, et al. **"A cross-layer approach to heterogeneity and reliability."** *Proceedings of the 7th IEEE/ACM international conference on Formal Methods and Models for Codesign.* IEEE Press, 2009.
-
-### University of Virginia, School of Engineering & Applied Science
-#### B.S. with Honors in Computer Science - May 2005<br/> Also completed requirements for B.A. in Cognitive Science
-Thesis: **“NJTopol: Adding Topology to Parallelized Neural Simulation”**  
-Publications: Levy, William B., et al. **"The formation of neural codes in the hippocampus: trace conditioning as a prototypical paradigm for studying the random recoding hypothesis."** *Biological cybernetics* 92.6 (2005): 409-426.
+# Aprotim Sanyal 
+ 
+jobs@aprotim.com   
+(310) 243-6230 
+ 
+Employment 
+------------ 
+### Edgecast - **Principal Software Engineer, SRE** - *April 2020 - July 2022* 
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts. 
+ 
+- **Built organization-wide best practices.** My primary responsibility as an experienced SRE was evaluating existing procedures and tools 
```

Comment:
- **Built organization-wide best practices.** Evaluated existing processes and tooling across EdgeCast, recommending and implementing changes in line with industry standards